robosat/osm/road.py

```diff
@@ -0,0 +1,143 @@
+import sys
+
+import math
+import osmium
+import geojson
+import shapely.geometry
+
+from robosat.osm.core import is_polygon
+
+
+class RoadHandler(osmium.SimpleHandler):
+    """Extracts road polygon features (visible in satellite imagery) from the map.
+    """
+
+    #  highway=* to discard because these features are not vislible in satellite imagery
```

Comment:
What about footways or steps in the list below? Or other types listed here https://wiki.openstreetmap.org/wiki/Key:highway#Values

Should we handle all tagged roads or only the major ones you are listing here?

Reply:
Okay - if someone later down the line comes along and wants to extract foot paths etc. we can always extend it later :+1: